src/rust/collections/mod.rs

```diff
@@ -4,6 +4,8 @@
 pub mod async_queue;
 pub mod async_value;
 pub mod intrusive;
+mod raw_array;
+pub mod ring;
```

Comment:
If you move this to all LibOses, you should remove it from the if statement that follows.